pkgs/development/python-modules/shiboken6/default.nix

```diff
@@ -35,7 +34,10 @@ llvmPackages.stdenv.mkDerivation rec {
   buildInputs = [
     llvmPackages.llvm
     llvmPackages.libclang
-    qt6.qtbase
+    python.pkgs.qt6.qtbase
+    python.pkgs.ninja-python
```

Comment:
```suggestion
    python.pkgs.ninja
```

Reply:
Fixed in 1df4a27001f8c57a639422468c70d2e7240f8349.